utils/updateFileApproval.ts

```diff
@@ -0,0 +1,36 @@
+import { Student } from "@prisma/client";
+import { mutate } from "swr";
+
+/**
+ * updateFileApproval updates a file's approval from an admin
+ * @param filePath
+ * @param email
+ * @param approval
+ * @returns deleted records
+ */
+export async function updateFileApproval(
+    filePath: string,
+    email: string,
+    approval: boolean | null,
+): Promise<Response> {
+    const updateData = {
+        email: email,
+        approval: approval,
+    };
+
+    const request = {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(updateData),
+    };
+    console.log(request.body);
+
+    const response = await fetch(
+        `/api/admin/update-${filePath}-status`,
+        request,
+    );
+
+    mutate(`/api/admin/update-${filePath}-status`);
```

Comment:
here, the intention of the mutate is inform a SWR hook that it's data has been mutated and needs to be refetched.

In our case, /api/user/[id]` route...
Either way this will need to be tested later.

```suggestion
    mutate(`/api/user/${id}`);
```